pkg/server/plugin/upstreamauthority/vault/vault_client.go

```diff
@@ -101,11 +101,11 @@ type Client struct {
 // SignCSRResponse includes certificates which are generates by Vault
 type SignCSRResponse struct {
 	// A certificate requested to sign
-	CertPEM string
-	// A certificate of CA(Vault)
 	CACertPEM string
+	// A certificate of CA(Vault)
+	UpstreamCACertPEM string
```

Comment:
The field names has been changed in this commit.
But if it is better or prefer to keep the original, please comment.